packages/x-charts/src/models/axis.ts

```diff
@@ -61,15 +61,32 @@ export interface ChartsAxisProps extends TickParams {
   /**
    * The font size of the axis ticks text.
    * @default 12
+   * @deprecated You can us `tickLabelStyle.fontSize` instead.
```

Comment:
```suggestion
   * @deprecated Consider using `tickLabelStyle.fontSize` instead.
```

Reply:
This led me to discover a regression in @mui/docs-infra.
Previously, such use cases used to be transformed into being wrapped in a `<code>` block, whereas now it is wrapped in a `span`. 🙈 
<img width="1344" alt="Screenshot 2023-10-18 at 17 37 45" src="https://github.com/mui/mui-x/assets/4941090/d194f46f-4b0a-4356-b43a-ee355e0ed96e">